python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/datahub_client.py

```diff
@@ -397,321 +442,108 @@ def upsert_athena_table(
-                entityType="dataset",
-                changeType=ChangeTypeClass.UPSERT,
-                entityUrn=dataset_urn,
-                aspect=tags_to_add,
-            )
-            self.graph.emit(event)
-
-        if data_product_metadata is not None:
-            data_product_urn = "urn:li:dataProduct:" + "".join(
-                data_product_metadata.name.split()
-            )
-            data_product_exists = self.graph.exists(entity_urn=data_product_urn)
-
-            if not data_product_exists:
-                data_product_urn = self.upsert_data_product(
-                    metadata=data_product_metadata
-                )
-
-            data_product_existing_properties = self.graph.get_aspect(
-                entity_urn=data_product_urn, aspect_type=DataProductPropertiesClass
-            )
-
-            data_product_association = DataProductAssociationClass(
-                destinationUrn=dataset_urn, sourceUrn=data_product_urn
-            )
-
-            if (
-                data_product_existing_properties is not None
-                and data_product_existing_properties.assets is not None
-            ):
-                assets = data_product_existing_properties.assets[::]
-                if data_product_association not in assets:
-                    assets.append(data_product_association)
-            else:
-                assets = [data_product_association]
-
-            data_product_properties = DataProductPropertiesClass(
-                description=data_product_existing_properties.description,
-                name=data_product_existing_properties.name,
-                customProperties=data_product_existing_properties.customProperties,
-                assets=assets,
-            )
-
-            metadata_event = MetadataChangeProposalWrapper(
-                entityType="dataproduct",
-                changeType=ChangeTypeClass.UPSERT,
-                entityUrn=data_product_urn,
-                aspect=data_product_properties,
-            )
-            self.graph.emit(metadata_event)
-
-        return dataset_urn
+        return database_urn
 
-    def search(
-        self,
-        query: str = "*",
-        count: int = 20,
-        page: str | None = None,
-        result_types: Sequence[ResultType] = (
-            ResultType.DATA_PRODUCT,
-            ResultType.TABLE,
-            ResultType.CHART,
-            ResultType.DATABASE,
-        ),
-        filters: Sequence[MultiSelectFilter] = (),
-        sort: SortOption | None = None,
-    ) -> SearchResponse:
-        """
-        Wraps the catalogue's search function.
-        """
-        return self.search_client.search(
-            query=query,
-            count=count,
-            page=page,
-            result_types=result_types,
-            filters=filters,
-            sort=sort,
-        )
+    def list_database_tables(self, urn: str, count: int) -> SearchResponse:
```

Comment:
Seems a bit weird that a `list` method returns a `SearchResponse` rather than a `list[Table]`. But I think we can leave this for now.